data-pipeline/config.py

```
@@ -50,14 +51,36 @@ class DatabaseConfig:
+        try:
+            result = urlparse(db_url)
+            return DatabaseConfig(
+                host=result.hostname,
+                port=result.port or 5432,
+                database=result.path.lstrip('/'),
+                username=result.username,
+                password=result.password,
+                ssl_mode="require" # Force SSL for remote
+            )
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)